irs-edc-client/src/main/java/org/eclipse/tractusx/irs/edc/client/policy/Constraint.java

```diff
@@ -39,13 +39,14 @@
 public class Constraint {
 
     @Schema(implementation = String.class, example = "string")
-    @JsonAlias({"odrl:leftOperand"})
+    @JsonAlias({ "odrl:leftOperand" })
     private String leftOperand;
     @JsonAlias("odrl:operator")
     @Schema
     private Operator operator;
     @Schema(implementation = String.class, example = "string")
-    @JsonProperty("odrl:rightOperand")
+    @JsonAlias("odrl:rightOperand")
+    @JsonProperty("rightOperand")
```

Comment:
In case you use JsonProperty you shall add this mechanism for all other attributes as well. odrl:operator an odrl:leftOperand a.s.o

Reply:
I'll remove JsonProperty, shouldn't be necessary at all